operator/CHANGELOG.md

```diff
@@ -18,6 +18,29 @@ and is generated by [Changie](https://github.com/miniscruff/changie).
 * The redpanda operator's helm chart has been merged into the operator itself.
 
   Going forward the chart's `version` and `appVersion` will always be equal.
+### Fixed
```

Comment:
These changes shouldn't be present in main. Running `task generate` should fix it up for you.